pkg/machines/components/vmDisksTab.jsx

```diff
@@ -54,14 +58,40 @@ const VmDiskCell = ({ value, id }) => {
     );
 };
 
+const readonlyChanged = (vm, diskTarget) => {
```

Comment:
Sure, talking about merging, the updateDiskReadonly and updateDiskShareable (or toggle...) could be merged too, since only difference is in different XML element name. What do you think?